### PR TITLE
new workflow exercise, changed IP usage for hosts node name

### PR DESCRIPTION
**Pull request by goetzrieger**

### CODE REVIEW

File: content/ansible-tower-getting-started/2-cred/_index.md [hunk 1]
```diff
@@ -43,21 +43,22 @@ node3 ansible_host=<IP>
...
+  - In the inventory view of the Tower web UI click on your **Workshop Inventory**
 
   - Click on  the **HOSTS** button
 
   - To the right click the ![plus](../../images/green_plus.png?classes=inline) button.
 
   - **HOST NAME:** node1
-
-  - **Variables:** Under the three dashes `---`, enter **ansible_host: 22.33.44.55** in a new line. Make sure to enter your specific IP address for your **node1** from the inventory looked up above, and note that the variable definition has a colon **:** and a space between the values, not an equal sign **=** like in the inventory file.
+  <!-- - **Variables:** Under the three dashes `---`, enter **ansible_host: 22.33.44.55** in a new line. Make sure to enter your specific IP address for your **node1** from the inventory looked up above, and note that the variable definition has a colon **:** and a space between the values, not an equal sign **=** like in the inventory file. -->
```
Comment (cbolz) [inline content/ansible-tower-getting-started/2-cred/_index.md]: is this supposed to be a multi line comment? I guess the <!-- was added by mistake?

File: content/ansible-tower-getting-started/6-workflows/_index.md [hunk 2]
```diff
@@ -19,29 +19,35 @@ In this lab you’ll learn how to setup a workflow.
...
 
-When there is a new Tomcat server to deploy, two things need to happen:
+When there is a new NodeJS-based website to deploy, two main steps need to happen:
 
-  - Tomcat needs to be installed, the firewall needs to be opened and Tomcat should get started.
+The web operations team has to:
 
-  - The most recent version of the web application needs to be deployed.
+  - Install and configure NodeJS to run as a service.
+  - An Apache instance needs to be installed and configured as proxy to pass requests for the NodeJS content to NodeJSAnd a lot of other steps might be needed, too. Like SELinux, firewall... you know the drill.
```
Comment (cbolz) [inline content/ansible-tower-getting-started/6-workflows/_index.md]: NodeJSAnd - typo?

File: content/ansible-tower-getting-started/2-cred/_index.md (hunk 1 above)
Comment (goetzrieger) [reply content/ansible-tower-getting-started/2-cred/_index.md]: Deleted

File: content/ansible-tower-getting-started/6-workflows/_index.md (hunk 2 above)
Comment (goetzrieger) [reply content/ansible-tower-getting-started/6-workflows/_index.md]: Yup

**Review by cbolz — APPROVED** [review]

**Pull request merged**